pkg/scheduler/framework/plugins/examples/multipoint/multipoint.go

```diff
@@ -63,7 +63,24 @@ func (mc CommunicatingPlugin) Reserve(ctx context.Context, state *framework.Cycl
 	return nil
 }
 
-// PreBind is the functions invoked by the framework at "prebind" extension point.
+// Unreserve is the function invoked by the framework at "unreserve" extension
```

Comment:
Unreserve is the function invoked by the framework when any error happens during "reserve" extension point or later.